pom.xml

```diff
@@ -86,24 +86,22 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <version>${junit.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.springframework</groupId>
-                <artifactId>spring-context</artifactId>
-                <version>${spring.version}</version>
+                <groupId>com.sphereon.ms</groupId>
```

Comment:
Not open-source!

Reply:
That's fixed now, but the question is whether this project should be updated. 
The old Java 8 artifact runs perfectly fine with the newer ms/JVM. (As long as  you have a java.xml.bind dependency in your project)

Reply:
No it is a lib open to anybody so it makes sense to have the JVM level as low as possible